### ISSUE TITLE
Configuring Latex cross-link text

### ISSUE THREAD

**Issue by takluyver** [takluyver/bookbook#10]
As of PR #7, the labels on Latex cross links look like: *Chapter 2* or *Section 2.1*.

How do we let people configure these? Command line options? Notebook metadata? Nbconvert config? A separate bookbook config file?

**Comment by ketch** [comment]
Ideally you might detect from the .tplx template file which latex documentclass is used, and format this accordingly.  I don't know if that information is easy to access, though.

**Comment by takluyver** [comment]
It should be easy enough to regex it out in typical cases, however Latex is flexible enough that making a properly general solution would be difficult. I'm not familiar enough with Latex to know how it would break, though.

**Comment by ketch** [comment]
Yes; on second thought, there are just too many possible custom document classes; for instance, in my own template file I'm using "SIAMGHbook2016".  We don't want to try to keep up with that kind of thing.
